### DISCUSSION TITLE
Vercel 502: INTERNAL_ROUTER_CANNOT_GET_PATH

### DISCUSSION THREAD

**Discussion by destroyer22719** [vercel/community#2530] [Bug Report]
Hello, I'm having issues with Vercel failing to load some files under `/_next/static/chunks/` and this is causing my nextjs app to not work on vercel even though it works fine locally. Certain chunks not loading is causing my application on https://foodflation.me to not work properly.

The source code is all on https://github.com/destroyer22719/FoodFlation/

![Screenshot from 2023-05-20 11-16-39](https://github.com/vercel/community/assets/65511595/d666b258-5dbd-47c3-bdc9-ff2d00640d73)
When I directly entered the link of the chunks I see this page:
![Screenshot from 2023-05-20 11-17-17](https://github.com/vercel/community/assets/65511595/dc5406fe-a8f7-4884-97bc-88078412e368)

**Comment by amyegan** [comment]
Thanks for reporting this. The incident has been resolved, but you can find the update log at https://www.vercel-status.com/incidents/mg87wqkq22nt
